packages/larva/lib/utils/renderSass.js

```diff
@@ -0,0 +1,19 @@
+const sass = require( 'node-sass' );
+const path = require( 'path' );
+
+module.exports = function renderSass( sassData ) {
+	return new Promise( ( resolve, reject ) => {
+		sass.render( {
+			data: sassData,
+			includePaths: [
+				path.resolve( '/Users/laraschenck/Sites/pmc/pmc-larva/packages/css-utilities/node_modules') // command should be run from same directory as node_modules
```

Comment:
Dynamically resolve local path

Reply:
Ah, that is a place holder – I have a scaffold for a function to get the path that I need to fill in still.